src/server_manager/web_app/app.ts

```diff
@@ -85,6 +85,18 @@ async function computeDefaultAccessKeyDataLimit(
   }
 }
 
+// Returns whether we should display a disclaimer for the updated feature metrics data collection
+// policy. We only show it if all the following conditions are true:
+//   - data limits feature is not already enabled
+//   - opt-in metrics are enabled
+//   - user has not previously seen the data limits help bubble
+//   - user has not previously seen this disclaimer
+function shouldShowFeatureMetricsDisclaimer(server: server.Server): boolean {
```

Comment:
Let's have a `hasConsentForFeatureMetrics` or something along those lines. I think it's easier to reason about than having to understand the flows. 

It should return `dataLimits-feature-collection-consent` || `dataLimitsHelpBubble-dismissed`. Notice it's independent of the server.

Then you can have
```js
view.showFeatureMetricsDisclaimer = 
    server.getMetricsEnabled() &&
    !server.getAccessKeyDataLimit() &&
    !this.hasConsentForFeatureMetrics()
```

Reply:
Done.